src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -913,6 +914,9 @@ impl AstDisplay for DropDatabaseStatement {
             f.write_str("IF EXISTS ");
         }
         f.write_node(&self.name);
+        if self.cascade {
+            f.write_str(" CASCADE");
+        }
```

Comment:
I'd pattern the printing of this after whatever other `CASCADE`/`RESTRICT` statements do, e.g. `DROP INDEX`. Think that has a different default, but should inform the style of printing default options.